draft-high-assurance-profile-oid4vc-sd-jwt-vc.md

```diff
@@ -91,82 +96,86 @@ Unless explicitly stated, all normative requirements apply to all participating
 |OID4VP| N/A |MUST|MUST|
 |OID4VCI| MUST|MUST|N/A|
 |SIOPv2|N/A|MUST|SHOULD|
-|SD-JWT|MUST|MUST|MUST|
+|VC-SD-JWT|MUST|MUST|MUST|
 
 # OpenID for Verifiable Credential Issuance
 
+Implementations of this profile:
+
 * MUST support both pre-auth code flow and authorization code flow.
 * MUST support SD-JWT VC profile as defined in OID4VCI specification.
 * MUST support sender-constrained Tokens using a mechanism as defined in [@!I-D.ietf-oauth-dpop].
 
 ## Credential Offer
 
 * Both Grant Types `authorization_code` and `urn:ietf:params:oauth:grant-type:pre-authorized_code` MUST be supported as defined in Section 4.1.1 in [@!OIDF.OID4VCI]
-* For Grant Type `authorization_code`, the Issuer MUST include a scope value. The wallet MUST use that value in the `scope` Authorization parameter. For `urn:ietf:params:oauth:grant-type:pre-authorized_code`, pre-authorized code is used by the issuer to identify the credential type.
+* For Grant Type `authorization_code`, the Issuer MUST include a scope value in order to allow the Wallet to identify the desired credential type. The wallet MUST use that value in the `scope` Authorization parameter. For Grant Type `urn:ietf:params:oauth:grant-type:pre-authorized_code`, the pre-authorized code is used by the issuer to identify the credential type(s). (pending OID4VCI PR#519)
 * As a way to invoke the Wallet, at least a custom URL scheme `haip://` MUST be supported. Implementations MAY support other ways to invoke the wallets as agreed by trust frameworks/ecosystems/jurisdictions, not limited to using other custom URL schemes.
 
-Note: Authorization Code flow does not require Credential Offer from the Issuer to the Wallet, and fits the scenario where the Wallet has preconfigured issuers interested in being promoted by wallet as entrance into a VC ecosystem will be motivated to implement it, might be easier to implement with existing libs and on top of existing implementations.
+Note: The Authorization Code flow does not require a Credential Offer from the Issuer to the Wallet. However, it is included in the feature set of the Credential Offer because it might be easier to implement with existing libs and on top of existing implementations than the pre-authorized code Grant Tye.
```

Comment:
```suggestion
Note: The Authorization Code flow does not require a Credential Offer from the Issuer to the Wallet. However, it is included in the feature set of the Credential Offer because it might be easier to implement with existing libraries and on top of existing implementations than the pre-authorized code Grant Type.
```